Doc/README.rst

```diff
@@ -7,7 +7,7 @@ available <https://docs.python.org/dev/download.html>`_.
 
 Documentation on authoring Python documentation, including information about
 both style and markup, is available in the "`Documenting Python
-<https://docs.python.org/devguide/documenting.html>`_" chapter of the
+<https://devguide.python.org/documenting.html>`_" chapter of the
```

Comment:
This can be dev.guide.python.org/documenting/ (without the .html)

Reply:
Will do